### PR TITLE
feat(test-case-switcher): easily switch between saved test cases loca…

### PR DESCRIPTION
**Pull request by Csmith246**
…lly for the template you're developing

Example GIF from Slider:

SliderTestCaseSwitcherExample

### CODE REVIEW

**Review by kellyhutchins — APPROVED** [review]

**Comment by Csmith246** [comment]
Will hold off until after the release to merge this, just in case

**Pull request merged**
